README.md

```diff
@@ -36,30 +39,23 @@ sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-12 120
 
 All of the required Python packages are installed in a Python Virtual Environment. The first step is to create the virtual environment and the second step is to load/activate the environment. The first step has to be run only once.
 
-Create the Python Virtual Environment:  
+Create the Python Virtual Environment on Ubuntu 22.04 LTS:  
 ```
 python3.11 -m venv .venv
 ```
 
-Load/activate the Python Virtual Environment: 
+Load/activate the Python Virtual Environment in Ubuntu 22.04 LTS: 
 ```
 source .venv/bin/activate
 ```
 
 ### VTK Setup
 
-This project requires VTK to be built manually in order to enable offscreen and headless rendering.
-
 Before you can build VTK, you need to initalize the VTK submodule.
 
 ```
 git submodule init
 git submodule update
-```
-
-Compile and install the VTK modules:
-
-```
```

Comment:
The readme from the experimental branch is more up to date. Please include those instructions.